### DISCUSSION TITLE
which files are in .git folder?

### DISCUSSION THREAD

**Discussion by codetechstar** [community/community#123718] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello, everyone!
I downloaded some of project and the size of the project is 500M but mostly size is .git folder's.
Is this kind of virus?
Please let me know the reason.
Thank you for your attention.

**Comment by Vinit-S-Patil** [comment]
Hi there! No need to worry, it's highly unlikely that the large size of the project is due to a virus. 

".git"  folder is where Git stores all the information about your project's history, including every version of every file, commit data, branches, tags, and more. This folder can grow in size over time, especially in projects with a long history or large files.

However, if you're concerned about the size or suspect any malicious activity, you can always run a virus scan on your system to ensure everything is safe.

**Comment by codetechstar** [comment]
Yeah, but the number of commits is below 20.
Then is it possible?

**Comment by iTechTop** [comment, marked as answer]
😉Hi there! Don't worry, it's highly unlikely that your project folder being 500MB, mostly comprised of the .git folder, indicates a virus. 

The .git folder is where Git, a version control system, stores all the information about the history of your project, including every change made to every file. This folder can grow quite large, especially for projects with a long history or large files. So, it's perfectly normal for the .git folder to be the majority of the project's size.

However, if you're concerned about potential viruses or malware, it's always a good idea to run a virus scan on your system just to be safe. But in this case, it's likely just the nature of Git's version control system.

**Reply by tak11173132** [reply]
2023-12-28 — Apache Subversion 1.14.3 Released [¶](https://subversion.apache.org/#news-20231228-1.14.3)
